src/main/java/com/github/jdramaix/client/HomePage.ui.xml

```diff
@@ -0,0 +1,54 @@
+<ui:UiBinder xmlns:ui='urn:ui:com.google.gwt.uibinder'
```

Comment:
Maybe we should not use UiBinder here and instead place these elements directly in the host page and get the references with Document.get().getElementById